### ISSUE TITLE
Any solution for mapping model to React interface

### ISSUE THREAD

**Issue by nvcken** [wimagguc/react-django-admin#1]
I think you has already though about this.
Have you any concept or solution yet , I am thinking some kinds likes generate React form component from Django model

**Comment by wimagguc** [comment]
Yes, that would be the ultimate goal. It would require a lot more work though, and there will be scores of corner-case components: I just used a modal-view for uploading files for example, which would be very difficult to write a generator for.

**Comment by nvcken** [comment]
Sound would take time to implement.
At my urgent case, I think the light way is building separate UI for customize form which model need likes ( dynamic json form ...)  then send via django rest API. What do you think ?

**Comment by wimagguc** [comment]
Sure. Feel free to go ahead :)

**Comment by nvcken** [comment]
would good ref https://github.com/jrief/django-angular
